src/main/java/com/minecolonies/coremod/entity/ai/citizen/herders/EntityAIWorkCowboy.java

```diff
@@ -0,0 +1,130 @@
+package com.minecolonies.coremod.entity.ai.citizen.herders;
+
+import com.minecolonies.api.util.InventoryUtils;
+import com.minecolonies.api.util.ItemStackUtils;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.minecolonies.api.util.ItemStackUtils'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)